catalog/transformer/catalog-transformer-xml-binding/src/main/resources/result.xsd

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:mc="urn:catalog:metacard" xmlns="urn:catalog:result"
+        xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="urn:catalog:result"
+        elementFormDefault="qualified" attributeFormDefault="unqualified"
+        version="1.0.0">
+
+	<!-- bring in other schemas -->
+    <xsd:import schemaLocation="metacard.xsd" namespace="mc"/>
+
+    <xsd:element name="result">
+		<xsd:complexType>
+			<xsd:sequence>
+				<xsd:element ref="mc:metacard" minOccurs="1" maxOccurs="1">
+					<xsd:annotation>
+						<xsd:documentation>A wrapper for the metacard schema that contains a relevance score and distance.
+						</xsd:documentation>
+					</xsd:annotation>
+				</xsd:element>
+				<xsd:element name="relevance" minOccurs="0">
+					<xsd:simpleType>
+						<xsd:annotation>
+							<xsd:documentation>A "relevance" score (typical range of 0-1, values over 1 are those that have been boosted) is used to specify a sort policy based on relevance.
+							</xsd:documentation>
+						</xsd:annotation>
+						<xsd:restriction base="xsd:double">
+							<xsd:minInclusive value="0"/>
+							<xsd:maxInclusive value="1"/>
```

Comment:
Maybe I'm rusty on my schemas. In the documentation you specify the possibility of values over 1, however the maxInclusive is 1 so doesn't that mean the schema prohibits having a value over 1?